tests/php/test-helpers.php

```diff
@@ -21,8 +21,11 @@ public function test_block_field() {
 		global $block_lab_attributes, $block_lab_config;
```

Comment:
I'm making it a priority to add more unit tests to the plugin. They might have prevented an issue like this.